### DISCUSSION TITLE
Hardware Token Solution

### DISCUSSION THREAD

**Discussion by SamdaniPashaa** [neos-sdi/adfsmfa#358] [Q&A] [answered]
Hello Neos-Sdi Team

Hope you are doing well
currently we are using Neos-SDI  ADFSMFA in our environment and now we are looking for hardware token solution.
Need your support/Guide us in this.

Additonally, we are based in Saudi Arabia and so, let us known if you have any local partner here to support us.

Regards,
Samdani Pasha

**Comment by SamdaniPashaa** [comment]
Hello Team,
could you let us known which hardware tokens are supports to proceed with testing and implementation.
I have check with DUO (Cisco) and Symantec, as per them their tokens are not compatible with other MFA solutions.


please let us known, as you already tested.


thank you

**Comment by SamdaniPashaa** [comment]
Hello Team,
Finally it worked, we tested below hardware token without need to make any additional configuration on Neos-SDI ADFS MFA console.

Name: Token2 Molto-1-i Multi-profile TOTP hardware token


Thank you

**Comment by redhook62** [comment]
happy for you, can you explain how you did it ?

**Comment by SamdaniPashaa** [comment, marked as answer]
Hi Redhook62

the current config on ADFS MFA is set as shown in the snip below

and to configureTOTP profile on the device (https://www.token2.com/shop/product/token2-molto-1-i-multi-profile-totp-hardware-token) using android/iphone are 

- android mobile: https://www.token2.com/site/page/token2-nfc-burner-for-molto1-android-app
- IPhone: https://www.token2.com/site/page/molto-1-i-nfc-burner-app-for-iphone


![TOTP Config](https://github.com/user-attachments/assets/78d859a6-f69e-4b86-ba02-9ec1604f2c1f)


regards,
Samdani pasha
